package.json

```diff
@@ -7,9 +7,11 @@
 		"start": "DISPLAY=\"${DISPLAY:=:0}\" ./node_modules/.bin/electron js/electron.js",
 		"start:dev": "DISPLAY=\"${DISPLAY:=:0}\" ./node_modules/.bin/electron js/electron.js dev",
 		"server": "node ./serveronly",
-		"install": "echo \"Installing vendor files ...\n\" && cd vendor && npm install --loglevel=error",
```

Comment:
so what happens now when someone executes "npm install" in the directory?

Reply:
same as before. Normal `npm install` is executed (with audit ...) and then fonts and vendor installs are done.